src/redisearch_rs/buffer/src/lib.rs

```diff
@@ -70,6 +83,11 @@ impl Buffer {
         self.0.offset == 0
     }
 
+    /// Resets the buffer's length to zero, effectively clearing it.
+    pub fn reset(&mut self) {
```

Comment:
Let's call it clear, given https://doc.rust-lang.org/std/vec/struct.Vec.html#method.clear